javascript/packages/orchestrator/src/network.ts

```diff
@@ -152,6 +153,7 @@ export class Network {
     // Cleanup all api instances
     for (const node of Object.values(this.nodesByName))
       node.apiInstance?.disconnect();
+    await destroyChainSpecProcesses();
     await this.client.destroyNamespace();
```

Comment:
To avoid errors.

```suggestion
    for (const node of Object.values(this.nodesByName)) {
          node.apiInstance?.disconnect();
    }
    await destroyChainSpecProcesses();
    await this.client.destroyNamespace();
```